doc/content/integrations/ifttt/node-red-setup/_index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Node-RED Setup"
+description: ""
+weight: 2
+---
+
+This section shows how to create a flow that will act like a mediator between {{% tts %}} and IFTTT. This flow will receive JSON messages from {{% tts %}} via Webhooks integration, extract the decoded payload and send it as a payload of a separate HTTP POST request to IFTTT.
+
+<!--more-->
+
+Run Node-RED and open a new flow by clicking the **+** button in the upper right.
+
+Place the **http in** node on the dashboard and double-click on it to configure it.
+
+Select **POST** as a **Method**.
+
+In the **URL** field enter the arbitrary path name, eg. `join` if you are enabling the integration for the `Join accept` messages. Keep in mind that this path name also needs to be used when creating a Webhook on {{% tts %}}.
```

Comment:
Please be consistent (I'm fine with any, as long as we're consistent)
 
join request / Join request / Join Request
join accept / Join accept / Join Accept